src/components/forms/summitdoc-form.js

```diff
@@ -93,16 +93,25 @@ class SummitDocForm extends React.Component {
     handleUploadFile(file) {
         let entity = {...this.state.entity};
 
-        entity.file_preview = file.preview;
-
-        this.setState({file: file, entity: entity});
+        if (entity.id) {
+            const formData = new FormData();
```

Comment:
could we move this to action and create a helper func
this is communication concern not UI concern

Reply:
done